CONTRIBUTING.md

```diff
@@ -82,7 +82,7 @@ Password: `P@ssw0rd`
 
 ## Test environment
 
-Test environment is running on dev.devadventcalendar.pl with automatic deployment from develop_uat branch.
+Test environment is running on https://dev.devadventcalendar.pl/ with automatic deployment from develop_uat branch.
```

Comment:
in markdown you can create links
[link_name](link)
so please update file

Reply:
created link with correct structure